src/delegate/delegate.ios.ts

```diff
@@ -242,7 +241,7 @@ export class TwilioAppDelegate extends UIResponder
 
     this.callInvite = callInvite;
 
-    this.reportIncomingCall(callInvite.from, callInvite.uuid);
+    this.reportIncomingCall(callInvite.from, callInvite.uuid, callInvite);
```

Comment:
If you pass the whole `callInvite` param, I think passing the `callInvite.from` and `callInvite.uuid` is redundant. Could you please pass only the `callInvite`one and taking the `from` and `uuid` from it in the `reportIncomingCall` method?

Reply:
Agreed 💯 i have removed the callInvite uuid and from parameters from reportIncomingCall. I think we should add a TODO to define the most common variables of the callInvite Variable 💯 i recall having trouble finding the appropriate docs from twilio with regards to type declarations. I have resolved this issue